### PR TITLE
feat: add post by rizzabh

### PR DESCRIPTION
**Pull request by rizzabh**
Tweeted about Astro.js as an author.

### CODE REVIEW

File: src/content/posts/learning-astro.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: ../../layouts/PostLayout.astro # Always use this layout, it's so the post gets properly styled
+title: The title of the post
```
Comment (WilliamDavidHarrison) [inline src/content/posts/learning-astro.md:3]: Please add a title.

File: src/content/posts/learning-astro.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+excerpt: A short description of the post
+author: your-author-username
```
Comment (WilliamDavidHarrison) [inline src/content/posts/learning-astro.md:5]: Please add your author username here.

File: src/content/posts/learning-astro.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+layout: ../../layouts/PostLayout.astro # Always use this layout, it's so the post gets properly styled
+title: The title of the post
+excerpt: A short description of the post
```
Comment (WilliamDavidHarrison) [inline src/content/posts/learning-astro.md:4]: Please add a short description of your post.

File: src/content/posts/learning-astro.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+author: your-author-username
+createdAt: Month Day, Year
```
Comment (WilliamDavidHarrison) [inline src/content/posts/learning-astro.md:6]: Please add the date here.

**Review by WilliamDavidHarrison — APPROVED** [review]

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
Sorry, I meant to request changes.

File: src/content/posts/learning-astro.md (hunk 4 above)
Comment (rizzabh) [reply src/content/posts/learning-astro.md]: Hey check the commit now sorry i overlooked that part :)

**Review by oyepriyansh — APPROVED** [review]
looks good to merge

**Pull request merged**